daemon/logger/logentries/logentries.go

```diff
@@ -38,32 +41,46 @@ func init() {
 func New(info logger.Info) (logger.Logger, error) {
 	logrus.WithField("container", info.ContainerID).
 		WithField("token", info.Config[token]).
+		WithField("line-only", info.Config[lineonly]).
 		Debug("logging driver logentries configured")
 
 	log, err := le_go.Connect(info.Config[token])
 	if err != nil {
 		return nil, err
 	}
+	var lineOnly bool
+	if "" == info.Config[lineonly] {
```

Comment:
```
if _, exists := info.Config[lineOnly]; exists {
  // do strconv
}
```

There is no need to explicitly set false.
I think we should error out if nothing is set, which will happen with `ParseBool`